src/network-manager/azext_network_manager/_params.py

```diff
@@ -536,3 +541,155 @@ def load_arguments(self, _):
         c.ignore('user_rule')
+
+    with self.argument_context('network manager connection create') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('network_manager_connection_name', options_list=['--name', '-n', '--connection-name'],
+                   type=str, help='The name of the network manager connection.')
+        c.argument('network_manager_id', options_list=['--network-manager', '--network-manager-id'], type=str,
+                   help='the name or id of the network manager.', validator=validate_network_manager)
+        c.argument('description', type=str, help='A description of the scope connection.')
+
+    with self.argument_context('network manager connection update') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('network_manager_connection_name', options_list=['--name', '-n', '--connection-name'],
+                   type=str, help='The name of the network manager connection.', id_part='name')
+        c.argument('network_manager_id', options_list=['--network-manager'], type=str,
+                   help='the name or id of the network manager.', validator=validate_network_manager)
+        c.argument('description', type=str, help='A description of the scope connection.')
+        c.ignore('parameters')
+
+    with self.argument_context('network manager connection list') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('top', type=int, help='An optional query parameter which specifies the maximum number of records to '
+                   'be returned by the server.')
+        c.argument('skip_token', type=str, help='SkipToken is only used if a previous operation returned a partial '
+                   'result. If a previous response contains a nextLink element, the value of the nextLink element will '
+                   'include a skipToken parameter that specifies a starting point to use for subsequent calls.')
+
+    with self.argument_context('network manager connection show') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('network_manager_connection_name', options_list=['--name', '-n', '--connection-name'],
+                   type=str, help='The name of the network manager connection.', id_part='name')
+
+    with self.argument_context('network manager connection delete') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('network_manager_connection_name', options_list=['--name', '-n', '--connection-name'],
+                   type=str, help='The name of the network manager connection.', id_part='name')
+
+    with self.argument_context('network manager management-group connection create') as c:
+        c.argument('resource_group_name', resource_group_name_type)
```

Comment:
there is no resource group id needed here, the resource id's follow
/providers/Microsoft.Network/managementGroups/{managementGroupId}/networkManagerConnections/{networkManagerConnectionName}

Reply:
Ok, I'll modify it.